editing/edit-with-branch-versioning/src/main/java/com/esri/samples/edit_with_branch_versioning/EditWithBranchVersioningController.java

```diff
@@ -283,18 +285,18 @@ private void switchVersion(String versionName) {
   }
 
   /**
-   * Select a feature if one exists where the user clicked
+   * Select a feature if one exists where the user clicked.
    *
    * @param point location where the user clicked
    */
-  private void selectFeature(Point2D point){
-    ListenableFuture<IdentifyLayerResult> results = mapView.identifyLayerAsync(featureLayer, point, 1, false);
-    results.addDoneListener(() -> {
+  private void selectFeature(Point2D point) {
+    ListenableFuture<IdentifyLayerResult> identifyLayerResultFuture = mapView.identifyLayerAsync(featureLayer, point, 1, false);
+    identifyLayerResultFuture.addDoneListener(() -> {
       try {
-        IdentifyLayerResult layer = results.get();
-        List<GeoElement> identified = layer.getElements();
-        if (!identified.isEmpty()) {
-          GeoElement element = identified.get(0);
+        var identifyLayerResult = identifyLayerResultFuture.get();
```

Comment:
Although we could use the predictive typing `var` here (it's more modern and sleek), since we're trying to demonstrate how the API works, lets declare this variable with the full type (as you had it before):

```suggestion
        IdentifyLayerResult identifyLayerResult = identifyLayerResultFuture.get();
```

That would help the users understand what the return type is of the `.get()`

Same would apply to line 299:
```
GeoElement element = identified.get(0);
```